docs/udales-namoptions-overview.md

```diff
@@ -1,132 +1,101 @@
-| ---- | ------- | --------------- | ----------- | ---- |
-| lfielddump | .false. | .true., .false. | Switch for instantaneous field output. | - |
-| tfielddump | 10000. | | Output time for fields. | s |
-| fieldvars | '' | Any of the given labels, several are seperated by a comma: u0,v0,w0,th,ql,qt,p0,s1,s2,s3,s4,s5 | Variable names of fields. | - |
-| tsample | 5. | | Sample time for statistics. | s |
-| tstatsdump | 10000. | | Output time for statistics. | s |
-| ltdump | .false. | .true., .false. | Switch to output time-averaged statistics. | - |
-| lydump | .false. | .true., .false. | Switch to output y-averaged statistics. *Only functional if x-direction is not parallelised.* | - |
-| lytdump | .false. | .true., .false. | Switch to output y- and time- averaged statistics. *Only functional if x-direction is not parallelised.* | - |
-| lxydump | .false. | .true., .false. | Switch to output x- and y- averaged statistics. | - |
-| lxytdump | .false. | .true., .false. | Switch to output x-, y- and time-averaged statistics. | - |
-| lslicedump | .false. | .true., .false. | Switch to output slices in the xy-plane. | - |
-| ltkedump | .false. | .true., .false. | *Not supported in the current version.* | - |
+Possible advection schemes:
 
-## Namelist NAMSUBGRID
+1 = 1st order upwind scheme
 
-| Name | Default | Possible values | Description | Unit |
-| ---- | ------- | --------------- | ----------- | ---- |
-| lvreman | .false. | .true., .false. | Switch for Vreman (2004) sub-grid scheme. | - |
-| c_vreman | 0.07 | | Model constant for Vreman scheme. | |
-| lbuoycorr | .false. | .true., .false. | Switch for buoyancy correlation in the Vreman scheme. | - |
-| loneeqn | .false. | .true., .false. | Switch for one-equation sub-grid scheme. | - |
-| ldelta | .false. | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
-| lmason | .false. | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
-| cf | 2.5 | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
-| cn | 0.76 | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
-| Rigc | 0.25 | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
-| Prandtl | 0.333 | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
-| lsmagorinsky | .false . | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
-| cs | -1 | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
-| nmason | 2 | | See [DALES](https://github.com/dalesteam/dales/blob/master/utils/doc/input/Namoptions.pdf). | |
+2 = 2nd order central difference scheme
 
-## Namelist NAMCHECKSIM
+7 = Kappa (flux limited) scheme. This scheme designed for quantities that should never become negative.
```

Comment:
Say that this is only possible to use for scalar concentrations - the other scalars (temperature and moisture) have to use 2nd order schemes.